app/src/main/java/engineering/reverse/ludumcomments/MainActivity.java

```diff
@@ -133,7 +191,7 @@ private void load() {
                             @Override
                             public void onResponse(JSONObject response) {
                                 try {
-                                    JSONArray jsonArray = response.getJSONArray(note);
+                                    JSONArray jsonArray = response.getJSONArray("note");
                                     SharedPreferences.Editor editor = prefs.edit();
```

Comment:
Now that we are not using quarkbackend, can you remove the logic for needing sharedpreferences for this altogether.